<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAchievements</class>
 <widget class="QWidget" name="FormAchievements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SteamAchievementsStatistic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="FrameFilter">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="LabelTotalPersent">
           <property name="text">
            <string notr="true">none/none (0%)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonUpdate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Обновить</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="FormReachedFilter" name="FilterMyProfile" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="CheckBoxFavorites">
           <property name="text">
            <string>Только избранное</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonGuides">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Руководства</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="LineEditNameAchievements">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="placeholderText">
            <string>Достижение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonFindAchievement">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="FormCategoriesTree" name="TreeWidgetCategories">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Категории</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="LabelGameOnline">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Ebrima</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>GameOnline:  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabelGameOnlineValue">
           <property name="font">
            <font>
             <family>Ebrima</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #42a9c6;</string>
           </property>
           <property name="text">
            <string notr="true">0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="TabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabStandart">
       <attribute name="title">
        <string> Свои достижения  </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="TableViewMyAchievements"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEditCategory">
       <attribute name="title">
        <string>  Управление категориями  </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="FormCategoriesEdit" name="CategoriesEdit" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCompare">
       <attribute name="title">
        <string>  Сравнение с друзьями </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="FormFriendsCompare" name="FriendsCompare" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ProgressBarGood" name="ProgressBarLoad">
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v/%m</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormCategoriesEdit</class>
   <extends>QWidget</extends>
   <header>forms/subForms/achievements/formcategoriesedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormReachedFilter</class>
   <extends>QWidget</extends>
   <header>forms/subForms/achievements/widgets/formreachedfilter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormFriendsCompare</class>
   <extends>QWidget</extends>
   <header>forms/subForms/achievements/formfriendscompare.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgressBarGood</class>
   <extends>QProgressBar</extends>
   <header>subWidgets/progressBars/progressbargood.h</header>
  </customwidget>
  <customwidget>
   <class>FormCategoriesTree</class>
   <extends>QTreeWidget</extends>
   <header>forms/subForms/achievements/formcategoriestree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
